src/style.css

```diff
@@ -0,0 +1,40 @@
+.posts-list{
```

Comment:
@jyotigautam leave spaces throughout the file after the property name `background-color:#ffffff;` should be `background-color: #ffffff;`.

Also, don't use hardcoded colour hex create a scss variable. Actually let's switch to scss going forward.